ecsmanage/management/commands/ecsmanage.py

```diff
@@ -0,0 +1,228 @@
+import boto3
+from django.core.management.base import BaseCommand, CommandError
+from django.conf import settings
+
+
+class Command(BaseCommand):
+    help = 'Run a one-off management command on an ECS cluster.'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '-e', '--env',
+            type=str,
+            default='default',
+            help=(
+                'Environment to run the task in, as defined in'
+                'ECSMANAGE_ENVIRONMENTS.'
+            )
+        )
+
+        parser.add_argument(
+            'cmd',
+            type=str,
+            nargs='+',
+            help="Command override for the ECS container (e.g. 'migrate')"
+        )
+
+    def handle(self, *args, **options):
+        """
+        Run the given command on the latest app CLI task definition and print
+        out a URL to view the status.
+        """
+        self.env = options['env']
+        cmd = options['cmd']
+
+        config = self.parse_config()
+
+        aws_region = config['AWS_REGION']
+
+        self.ecs_client = boto3.client('ecs', region_name=aws_region)
+        self.ec2_client = boto3.client('ec2', region_name=aws_region)
+
+        task_def_arn = self.get_task_def(config['TASK_DEFINITION_NAME'])
+        security_group_id = self.get_security_group(
+            config['SECURITY_GROUP_TAGS']
+        )
+        subnet_id = self.get_subnet(config['SUBNET_TAGS'])
+
+        task_id = self.run_task(config,
+                                task_def_arn,
+                                security_group_id,
+                                subnet_id,
+                                cmd)
+
+        cluster_name = config['CLUSTER_NAME']
+
+        url = (
+            f'https://console.aws.amazon.com/ecs/home?region={aws_region}#'
+            f'/clusters/{cluster_name}/tasks/{task_id}/details'
+        )
+
+        self.stdout.write(self.style.SUCCESS('Task started! View here:\n'))
```

Comment:
For I/O buffering purposes in CloudWatch Logs, it might be beneficial to make both of these a single `self.stdout.write` call.